README.md

```diff
@@ -319,6 +319,8 @@ limited to the default appearance - Skim supports comprehensive customization of
 --color=[BASE_SCHEME][,COLOR:ANSI]
 ```
 
+Skim also respects the `NO_COLOR` environment variable.
```

Comment:
This needs a little more details, maybe link to https://no-color.org/ and precise the usage